lib/tower_cli/resources/project.py

```diff
@@ -98,7 +98,7 @@ def create(self, organization=None, monitor=False, timeout=None,
             org_resource._assoc('projects', org_pk, project_id)
 
         # if the monitor flag is set, wait for the SCM to update
-        if monitor:
+        if monitor and answer['changed']:
```

Comment:
The parent method should alway give a dictionary with `changed` in it (after having checked the code), but it would still be better to do ` and answer.get('changed', False)` here, in order to alleviate any of those concerns.

But that's my only comment. Nice work tracking this down!